manifests/d/Devolutions/RemoteDesktopManagerAgent/2024.2.15.0/Devolutions.RemoteDesktopManagerAgent.locale.en-US.yaml

```diff
@@ -0,0 +1,28 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Devolutions.RemoteDesktopManagerAgent
+PackageVersion: 2024.2.15.0
+PackageLocale: en-US
+Publisher: Devolutions inc.
+PublisherUrl: https://devolutions.net/
+PublisherSupportUrl: https://devolutions.net/support
+PrivacyUrl: https://devolutions.net/legal/privacy
+Author: Devolutions
+PackageName: Remote Desktop Manager Agent
+PackageUrl: 
```

Comment:
This shouldn't be empty. Either there needs to be a valid URL or the field to be removed entirely